### DISCUSSION TITLE
Can Developers/Students get faster access to Github Copilot

### DISCUSSION THREAD

**Discussion by hunterz-killer** [community/community#11775] [Copilot Conversations] [answered]
Hi Guys,

Can we a higher priority for Developers/Students get faster access to Github Copilot, So that it would be highly helpful.

**Comment by davecheney** [comment, marked as answer]
Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060

**Reply by hunterz-killer** [reply]
thanks


Is there any possibilities to get faster approval

**Reply by davecheney** [reply]
Ask on https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
